internal/docker/start.go

```diff
@@ -93,6 +95,14 @@ func (s Start) Run(ctx context.Context, logs io.Writer, name, command string) (s
 		env = append(env, "VCAP_SERVICES={}")
 	}
 
+	if s.startCommand != "" {
+		command = s.startCommand
+	}
+
+	if command == "" {
+		return "", "", fmt.Errorf("error: Start command not specified")
```

Comment:
Should this error message line up to match what CF returns? Something like we assert against here: https://github.com/cloudfoundry/python-buildpack/blob/e263795d6838e8bd9033c87bd0118d6e6f165bee/src/python/integration/miscellaneous_test.go#L152-L153

Reply:
I think one thing is the error and the logs. In the logs indeed we have the lines:

```
No start command specified by buildpack or via Procfile.
App will not start unless a command is provided at runtime.
```

For both Docker and CF, the actual errors differ a little bit from CF and Docker:
cf
```
Error staging application: Staging error: Start command not specified
```

Docker
```
error: failed to run start phase: error: Start command not specified
```

Reply:
Ok. I'm satisfied that at least the last part of the error message matches.